libs/mocks/src/asset_registry.rs

```diff
@@ -4,7 +4,7 @@ pub mod pallet {
 	use mock_builder::{execute_call, register_call};
 	use orml_traits::asset_registry::{AssetMetadata, Inspect, Mutate};
 	use sp_std::fmt::Debug;
-	use staging_xcm::{v3::prelude::MultiLocation, VersionedMultiLocation};
+	use staging_xcm::{v4::Location, VersionedLocation};
```

Comment:
:warn: The asset registry's API has been changed to V4. Maybe we need to migrate our asset registry to transform locations to V4.

Not sure if this is mandatory because the locations are behind a `VersionedLocation` allowing to exists different versions. But if a several places in our code we expect they content an specific version, so if we do not migrate we need to check for different versions in those places.